api/osb/osb_controller.go

```diff
@@ -123,15 +126,32 @@ func (c *Controller) proxy(r *web.Request, logger *logrus.Entry, broker *types.S
 
 	proxy.ServeHTTP(recorder, modifiedRequest)
 
-	respBody, err := ioutil.ReadAll(recorder.Body)
+	brokerResponseBody, err := ioutil.ReadAll(recorder.Body)
 	if err != nil {
 		return nil, err
 	}
 
+	responseBody := brokerResponseBody
+
+	if !gjson.ValidBytes(brokerResponseBody) {
+		recorder.Header().Set("Content-Type", "application/json")
```

Comment:
This check is because I don't want to always overwrite the Content-Type header sent by the broker. Not sure if in all cases should be application/json...